bolt-sidecar/src/config/mod.rs

```diff
@@ -180,8 +183,11 @@ impl TryFrom<Opts> for Config {
             config.limits.max_committed_gas_per_slot = max_committed_gas;
         }
 
-        config.commit_boost_url =
-            opts.signing.commit_boost_url.as_ref().map(|url| Url::parse(url)).transpose()?;
+        config.commit_boost_address = opts
+            .signing
+            .commit_boost_url
+            .as_ref()
+            .map(|url| url.strip_prefix("http://").unwrap_or(url).to_string());
```

Comment:
We will have to remove the "http://" beacuse apparently the commit-boost new impl takes only the _address_ https://github.com/Commit-Boost/commit-boost-client/blob/828b10811f125bcc8ab5057df033aa29286d6130/crates/common/src/commit/client.rs#L31

I thought its cleaner to take the url as input, as in all the env variables and just remove the http here.

Reply:
To cover for cases where users provide an https address as well, we can parse the string into a `reqwest::Url` and then extract the `SocketAddr`